src/NetAnalyzers/RulesMissingDocumentation.md

```diff
@@ -4,9 +4,13 @@ Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
 CA1418 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1418> | Use valid platform string |
 CA1419 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1419> | Provide a public parameterless constructor for concrete types derived from 'System.Runtime.InteropServices.SafeHandle' |
+CA1727 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1727> | Use PascalCase for log message tokens |
 CA1839 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1839> | Use 'Environment.ProcessPath' |
 CA1840 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1840> | Use 'Environment.CurrentManagedThreadId' |
 CA1842 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1842> | Do not use 'WhenAll' with a single task |
 CA1843 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1843> | Do not use 'WaitAll' with a single task |
-CA2251 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2251> | Use 'string.Equals' |
```

Comment:
Mistakenly removed?

Reply:
no, the 

> build.cmd -pack 

script removes it

Reply:
@jmarolf calling out to check if it is ok to delete this line. Seems reasonable since the docs page is not missing anymore:

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2251